content/version/1/4/code-of-conduct.id.md

```diff
@@ -0,0 +1,68 @@
++++
+version = "1.4"
+aliases = ["/version/1/4"]
++++
+
+# Kode Etik Contributor Covenant
+
+## Janji Kami
+
+Dalam minat kami untuk membuat lingkungan yang terbuka dan ramah, kami sebagai 
+kontributor dan pengelola berjanji untuk membuat partisipasi dalam proyek kami
+dan komunitas kami bebas dari pelecehan untuk semua orang, tanap memandang usia,
+ukuran tubuh, disabilitas, etnik, identitas gender dan ekspresinya, tingkat pengalaman, pendidikan, status sosial ekonomi, nationalitas, penampilan pribadi, ras, agama, atau identitas dan orientasi seksual.
```

Comment:
I think you should use 'kelompok etnis' since both 'etnis' and 'etnik' are adjectives according to KBBI. and please change to 'kewarganegaraan' instead of 'nationalitas'.